spreading_test.txt

```diff
@@ -91,13 +106,26 @@ MovementModel.worldSize = 4500, 3400
 MovementModel.warmup = 1000
 
 # how many reports to load
-Report.nrofReports = 1
+Report.nrofReports = 2
 # length of the warm up period (simulated seconds)
 Report.warmup = 0
 # default directory of reports (can be overridden per Report with output setting)
 Report.reportDir = reports/
 # Report classes to load
 Report.report1 = InfectionReport
+# Configuration of parameters for the infection report
+# for each (distance, factor) pair, one entry in factor_distance, distances, in ascending order
+Report.factor_distance = 1.0, 1.0, 1.0, 0.2, 0.001
+Report.distances = 0.0, 0.5, 1.0, 1.5, 3.0
+# infection rate between 0.0001 and 0.001 seems to produce believable results
+Report.infection_rate = 0.0025
+# four different factors: indoor, main stage, shisha bar and outside
+Report.factor_location = 1.0, 5.0, 10.0, 0.5
+# three values each for sender and receiver: unvaccinated, vaccinated, booster
+Report.factor_sender = 1.0, 0.9, 0.8
+Report.factor_receiver = 1.0, 0.8, 0.7
```

Comment:
Auf was basieren die Faktoren? Vllt noch eine Quelle in den Kommentaren

Reply:
Die Quellen klingen auf jeden Fall gut! Habe gesehen, dass du sie auch im InfectionReport auch explizit genannt hast. Deshalb würde ich einfach noch einen Kommentar hinzufügen, der darauf verweist